src/components/Newsletter.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { MdOutlineMarkEmailRead } from 'react-icons/md';
+
+export const Newsletter = () => {
+  return (
+    <section className='sm:my-10'>
+      <div className='mx-auto px-4 sm:px-6 lg:px-8'>
+        <div className='relative -mx-4 overflow-hidden bg-slate-300 py-16 px-4 dark:bg-slate-800 sm:px-6 md:mx-0 md:rounded-3xl md:px-16 xl:px-24'>
+          <div className='relative mx-auto grid max-w-2xl grid-cols-1 gap-x-32 gap-y-14 text-gray-900 dark:text-slate-200 xl:max-w-none xl:grid-cols-2'>
+            <div>
+              <h2 className='text-4xl font-extrabold tracking-tight  sm:text-4xl'>
+                <span>Rimaniamo in contatto?</span>
+                <br />
+                <span className='bg-gradient-to-br from-primary-dark to-primary-light dark:from-primary dark:to-primary-lighter bg-clip-text text-transparent'>
+                  Tieniti sempre aggiornato.
+                </span>
+              </h2>
+              <p className='mt-2 text-lg'>
+                Organizziamo eventi gratuiti incentrati sul mondo tech.
+                Iscrivendoti non rischierai di rimanerne all&apos;oscuro!
+              </p>
+            </div>
+            <form
+              action='https://github.us21.list-manage.com/subscribe/post?u=08bff1fa2d8bd95bf693be2ab&amp;id=8489b0a8a1&amp;f_id=00bbf0e6f0'
+              method='post'
+              id='mc-embedded-subscribe-form'
+              name='mc-embedded-subscribe-form'
+              target='_self'
+            >
+              <h3 className='text-lg font-semibold tracking-tight text-primary-dark dark:text-primary-light'>
+                Iscriviti alla nostra newsletter
+                <span aria-hidden='true'>↓</span>
+              </h3>
+              <span className='hidden'>
+                {/* /* real people should not fill this in and expect good things - do
+               not remove this or risk form bot signups */}
+                <input
+                  type='text'
+                  name='b_08bff1fa2d8bd95bf693be2ab_8489b0a8a1'
+                  tabIndex={-1}
+                  value=''
```

Comment:
```suggestion
                  value=''
                  readOnly={true}
```
this is only to remove a warning produced by Next on compiling step

> Warning: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
>     at input
>     at span
>     at form
>     at div
>     at div
>     at div
>     at section
>     at Newsletter
>     at main
>     at Home (webpack-internal:///./src/pages/index.tsx:34:25)
>     at MdxComponentsProvider (webpack-internal:///./src/context/mdxContext.tsx:14:34)
>     at App (webpack-internal:///./src/pages/_app.tsx:19:16)
>     at StyleRegistry (/Users/matteo/Documents/Mine/Web/latina-in-tech.github.io/node_modules/styled-jsx/dist/index/index.js:449:36)
>     at eJ (/Users/matteo/Documents/Mine/Web/latina-in-tech.github.io/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:8:22325)
>     at eI (/Users/matteo/Documents/Mine/Web/latina-in-tech.github.io/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:17:1762)
>     at eH (/Users/matteo/Documents/Mine/Web/latina-in-tech.github.io/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:17:3085)
>     at div
>     at eQ (/Users/matteo/Documents/Mine/Web/latina-in-tech.github.io/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:26:742)